src/App.jsx

```diff
@@ -1,24 +1,17 @@
-
-import './App.css'
+import './App.css' 
+import TodoList from './TodoList.jsx'  // Added ./ at the beginning
```

Comment:
Remember to always format you code. Format as you code and format before submitting a PR. You will start noticing the difficulty in reading unformatted code as your codebase grows. IDEs usually have a handy hotkey combo or you can find it in the dropdown menu. (Or maybe a third part library has been installed)